docs/docs/behaviors/macros.md

```diff
@@ -134,6 +134,12 @@ bindings
     ;
 ```
 
+### Behavior Queue Limit
+
+The default queue limit is 64. However, some actions require multiple events (e.g. `&kp` requires `key press` and `key release` events). As a result, the effective number of actions can be less than 64.
+
+You can change the limit by adding `CONFIG_ZMK_BEHAVIORS_QUEUE_SIZE=n` to your `.conf` file. For example, you can add `CONFIG_ZMK_BEHAVIORS_QUEUE_SIZE=512` if your macro outputs more than 200 characters.
```

Comment:
```suggestion
To prevent issues with longer macros, you can change the size of this queue via the `CONFIG_ZMK_BEHAVIORS_QUEUE_SIZE` setting in your configuration, [typically through your `.conf` file](../config/index.md). For example, `CONFIG_ZMK_BEHAVIORS_QUEUE_SIZE=512` would allow your macro to type about 256 characters.
```

I added a link to config documentation to keep this generic, but we need to double check the link works correctly.